doc/tutorial/Tutorial.tex

```diff
@@ -664,10 +664,10 @@ \subsection{Existential quantification}
 Lemma refl_if : forall x : D, (exists y, R x y) -> R x x.
 \end{coq_example}
 
-Remark that the hypotheses which are local to the currently opened sections
+The hypotheses that are local to the currently opened sections
 are listed as local hypotheses to the current goals.
-The rationale is that these hypotheses are going to be discharged, as we
-shall see, when we shall close the corresponding sections.
+The rationale for this is that these hypotheses are going to be discharged, as
```

Comment:
Clarity. Otherwise, the sentence begs questions like, "Which rationale? The rationale for what? Is 'rationale' a technical term in Coq that has a specific meaning, like 'hypothesis' and 'goal'?"

This change, like all the edits here, is not crucial. Like all of them, it is intended to improve readability. That requires reducing the effort for the reader, by reducing the number of unnecessary and distracting questions that might arise in the reader's mind, even if the reader can probably resolve them without much trouble.

"What does 'Remark that' mean? Did the author mean 'Note that'? I will tentatively assume so for now, but will keep it in mind in case I find evidence to the contrary." is one example of such a question and resolution, and is addressed elsewhere in this PR. "The rationale for what? (Etc.)" is another example.

Anyhow, I think `s/The rationale is that/This is because/` would have been a better substitution, so I will make that change in a follow-up commit.